timely_beliefs/beliefs/utils.py

```diff
@@ -1054,22 +1054,57 @@ def meta_repr(
     )
 
 
+def convert_to_instantaneous(
```

Comment:
Isn't this function converting a instantaneous events (i.e. event_resolution = 0) to an equivalent non-instantaneous events  (i.e. event_resolution > 0)? I mean, in the end, more than converting this function is creating an equivalent representation.

Reply:
It's doing the opposite, from non-instantaneous to instantaneous, but I see how it's converting to an equivalent representation.